_drafts/community-group-overview.md

```diff
@@ -1,48 +1,55 @@
 
-- **Python coding group**: Discusses methods of programming in Python and troubleshoots code of participants. Python coders of all levels of expertise are welcome. See the #python-user-comm channel on The Coop Communities Slack for more information.
+- **Python User Group**: The Python User Group meets weekly to discuss programming, data analysis, and troubleshooting in python as well as dedicate one meeting a month to general software development; topics will be announced ahead of time. Bring your lunch, code, questions, and thoughts! Python coders of all levels of expertise are welcome. Contact the Coop (coophelp at fredhutch.org) for more information. Additionally, check out the #python-user-comm channel/discussion within the [Coop Communities Slack](https://fredhutch.github.io/coop/community/slack-new/) for more information and our other training and community resources at [Training, Community and Finding Help](https://sciwiki.fredhutch.org/scicomputing/reference_training/). Current meeting schedules and locations are available on the [Google calendar](https://fredhutch.github.io/FHBig/calendar/).
+When & where: every Friday from noon to 1 pm in the Coop Lab (M1-B406, Arnold).
 
-- **R coding group**: Discusses R packages, R code and troubleshoots code of participants. R coders of all levels of expertise are welcome. See the #r-user-comm channel on The Coop Communities Slack for more information.
-
-- **Software design group**: Discusses issues related to software engineering and interface design and invites speakers on specific topics and participants to share their coding projects. This group is not focused on a specific programming language, but rather on issues common across software projects. See the #software-engineers channel on The Coop Communities Slack for more information.
+- **R Coding Group**: The R User Group meets weekly to discuss programming, data analysis, and troubleshooting in R. Bring your lunch, code, questions, and thoughts! R coders of all levels of expertise are welcome. Contact the Coop (coophelp at fredhutch.org) for more information. Additionally, check out the #r-user-comm channel/discussion within the [Coop Communities Slack](https://fredhutch.github.io/coop/community/slack-new/) for more information and our other training and community resources at [Training, Community and Finding Help](https://sciwiki.fredhutch.org/scicomputing/reference_training/). Note: If you do not have a laptop available to you we still want you to come!! You can email code privately ahead of each event to Amy Paguirigan (apaguiri at fredhutch.org). Current meeting schedules and locations are available on the [Google calendar](https://fredhutch.github.io/FHBig/calendar/). When & where: every Thursday from noon to 1 pm in the Coop Lab (M1-B406, Arnold).
 
 #### How to start one
```

Comment:
Requesting a new community group

Reply:
adding and flushed out how to start one section